File: task3.js
//Task-3: Ədədlərdən ibarət olan bir array verilib həmin arraydə olan ən böyük ədədi ekrana yazdıran alqoritm yazın.
let ourArray = [-17,-11,-3,-6,-5];
let maxNumber = ourArray[0];
for(let i=1;i<ourArray.length;i++){
   if(ourArray[i]>maxNumber){
    maxNumber = ourArray[i];}
}
console.log(maxNumber);

File: task3.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Task-3: Ədədlərdən ibarət olan bir array verilib həmin arraydə olan ən böyük ədədi ekrana yazdıran alqoritm yazın.
            int[] ourArray = {-1,-2,-3};
            int maxNumber = ourArray[0];
            for (int i = 1; i < ourArray.Length; i++)
            {
                if (ourArray[i]>maxNumber)
                {
                    maxNumber = ourArray[i];
                }
            }
            Console.WriteLine(maxNumber);

        }
    }
}
